### PR TITLE
Fix Configuration for Xamarin.Android.sln

### PR DESCRIPTION
**Pull request by dellis1972**
If you loaded the .sln in Xamarin studio it defaults to the
`Debug` configuration. As a result the Java.Interop project is
NOT built as part of that configuration. This is because it was
looking for a Mapping that did not exist in that project.

While it didn't stop the build, it did looks odd and might cause
confusion for users.

Other configurations have also been updated to make sure all the
projects are building the correct projects

### CODE REVIEW

**Comment by jonpryor** [comment]
How was the configuration "broken"? What IDE reported warnings, and what were those warnings?

**Comment by dellis1972** [comment]
@jonpryor updated with comments :)

**Comment by jonpryor** [comment]
> As a result the Java.Interop project is NOT built as part of that configuration.

Java.Interop *isn't supposed to be built* in the Debug configuration. It shouldn't be built in *any* Xamarin.Android configuration...*except* by [Mono.Android.targets](https://github.com/xamarin/xamarin-android/blob/39c5905/src/Mono.Android/Mono.Android.targets#L7-KL10) as a `BeforeResolveReferences` target, which ensures that the correct Java.Interop configuration is built.

(The reason/rationale for this is that the original Java.Interop Debug configuration was "green field let's reimplement everything!", and when Cycle 7 had the opportunity for integration I didn't want to bring in *everything* -- it would be too destabilizing -- so Java.Interop got the `XAIntegrationDebug` and `XAIntegrationRelease` configurations, for use with Xamarin.Android. Perhaps now that should be *reversed*, with the Java.Interop Debug/Release configurations being for Xamarin.Android use, and some other configuration name for "let's include the kitchen sink!" For now, though, we have what we have.)

File: Xamarin.Android.sln [hunk 1]
```diff
@@ -112,6 +112,7 @@ Global
 		{94BD81F7-B06F-4295-9636-F8A3B6BDC762}.XAIntegrationDebug|Any CPU.Build.0 = XAIntegrationDebug|Any CPU
 		{94BD81F7-B06F-4295-9636-F8A3B6BDC762}.XAIntegrationRelease|Any CPU.ActiveCfg = XAIntegrationRelease|Any CPU
 		{94BD81F7-B06F-4295-9636-F8A3B6BDC762}.XAIntegrationRelease|Any CPU.Build.0 = XAIntegrationRelease|Any CPU
+		{94BD81F7-B06F-4295-9636-F8A3B6BDC762}.Debug|AnyCPU.Build.0 = Debug|Any CPU
```
Comment (jonpryor) [inline Xamarin.Android.sln]: As per previous commit, this line shouldn't be added.

**Pull request closed without merging**